pdm/cli/options.py

```diff
@@ -230,6 +250,14 @@ def no_isolation_callback(
     help="Save minimum version specifiers",
 )
 
+skip_option = Option(
+    "-k",
+    "--skip",
+    dest="skip",
+    action=split_lists(","),
+    help="Skip some tasks and/or hooks",
```

Comment:
Read from env var `PDM_SKIP_HOOKS` if the option is not given?

Reply:
Done 👍🏼 
It's not easily testable as `Option.default` is set once on parsing and can't change during tests but given I have to process the splitted list, I tested the method used to generate the `default`.